wasm/main.py

```diff
@@ -4599,11 +4580,12 @@ def module_exports(module):
     if len(exportstar) != len(externtypeprimestar):
         raise Exception("TODO: proper error message")
 
+    # TODO: this code path may not be excercised. Verify.
```

Comment:
:+1:

Reply:
Not that it matters really but...
```suggestion
    # TODO: this code path may not be exercised. Verify.
```